src/current/_includes/v23.1/sidebar-data/cloud-deployments.json

```diff
@@ -227,7 +227,7 @@
                         ]
                     },
                     {
-                        "title": "Manage Console Access",
+                        "title": "Managing Users, Roles, and Service Accounts",
```

Comment:
Do you want to keep this as imperative like the other titles, that is, "Manage Users, Roles, and Service Accounts"?

Reply:
+1